aio/content/examples/http/src/app/config/config.service.ts

```diff
@@ -82,8 +82,9 @@ export class ConfigService {
         `Backend returned code ${error.status}, body was: `, error.error);
     }
     // Return an observable with a user-facing error message.
-    return throwError(
-      'Something bad happened; please try again later.');
+    return throwError(() => {
+      new Error('Something bad happened; please try again later.');
+    });
```

Comment:
The error should be returned:

```suggestion
    return throwError(() => {
      return new Error('Something bad happened; please try again later.');
    });
```

This should fix the test 🙂

Reply:
:) That'll do it!